diff --git a/node_modules/@corbe30/fortune-excel/dist/ToFortuneSheet/FortuneSheet.js b/node_modules/@corbe30/fortune-excel/dist/ToFortuneSheet/FortuneSheet.js
index c7c2efc..2160fb5 100644
--- a/node_modules/@corbe30/fortune-excel/dist/ToFortuneSheet/FortuneSheet.js
+++ b/node_modules/@corbe30/fortune-excel/dist/ToFortuneSheet/FortuneSheet.js
@@ -516,7 +516,7 @@ var FortuneSheet = /** @class */ (function (_super) {
             var _hintShow = _hint ? true : false;
             var matchType = constant_1.COMMON_TYPE2.includes(_type) ? "common" : _type;
             _type2 = operator
-                ? constant_1.DATA_VERIFICATION_TYPE2_MAP[matchType][operator]
+                ? (constant_1.DATA_VERIFICATION_TYPE2_MAP[matchType] && constant_1.DATA_VERIFICATION_TYPE2_MAP[matchType][operator]) || "bw"
                 : "bw";
             // mobile phone number processing
             if (_type === "text_content" &&
